src/parsing/parser.ts

```diff
@@ -232,7 +235,8 @@ export function parseMappedTrackerSyntax(
 }
 
 function encodeTrackerData(valueSet: any[], keyType: string): any[] {
-  const values: any[] = valueSet.map((val) => {
+  const values: any[] = [];
+  valueSet.map((val) => {
```

Comment:
Map returns the inner function response to a new array so `const values` can be assigned directly from the map, andf the inner function can return the encoded value instead of pushing to `values`